### PR TITLE
not_finished

### PR DESCRIPTION
**Pull request by liliya-dev**
- [DEMO LINK](https://liliya-dev.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://liliya-dev.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +10,33 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Search bar airbnb</h1>
+  <div>
+    <form data-qa="big">
+      <div class="box_big">
+        <label for="big-search">
```
Comment (irenhh) [inline src/index.html:16]: for attribute is not necessary here

**Comment by irenhh** [comment]
image
have you checked the demo link?

File: src/index.html (hunk 1 above)
Comment (liliya-dev) [reply src/index.html]: > image
> have you checked the demo link?
yes, it seems it works
https://prnt.sc/s53ca6

Comment (liliya-dev) [reply src/index.html]: https://prnt.sc/s53djp

**Comment by irenhh** [comment]
image
you changed fonts only for Cyrillic alphabet

**Comment by liliya-dev** [comment]
> image
> you changed fonts only for Cyrillic alphabet
no, check
https://prnt.sc/s553f1

**Comment by liliya-dev** [comment]
> image
> you changed fonts only for Cyrillic alphabet

no, check
https://prnt.sc/s5540b

**Pull request closed without merging**